source/obox-man/doveadm-fscache.1.rst.in

```diff
@@ -0,0 +1,57 @@
+===============
+doveadm-fscache
+===============
+
+:Date:   2018-12-11
```

Comment:
Remove this.  We don't need a date here, and I have no idea what this even symbolizes.  (And even if we needed a date, it should be dynamically generated.)